java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture4/tasks/_3/Number3.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture4.tasks._3;
+
+import java.util.Scanner;
+import java.util.function.*;
+/*
+Create a functional interface StringModifier with a method modify that accepts a string and returns a modified string.
+In the main class create a method that accepts a string value and a lambda expression,
+applies the lambda to the value and outputs the result.
+In the main method create two lambda expressions that implement the interface:
+one should convert all characters to the lower case,
+second - to the upper case.
+Both lambdas should prepend a prefix “modified: ” before the modified values(e.g., “test” -> “modified: TEST”).
+Call the defined method with some string value and both lambdas.
+Call the same method with two method references that just convert all characters to the lower and upper case.
+ */
+
+public class Number3 {
+    public static void main(String[] args) {
+
+        String stringOne = "hElLo";
+        Function<String, String> function = string -> string + string;
+        Number3 number3 = new Number3();
+        number3.concatenation(stringOne, function);
+
+        //MARK: UPPERCASE AND lowercase LAMBDAS
+        StringModifier upperCase = string -> ('"' + string + '"' + " -> modified: " + '"' + string.toUpperCase() + '"');
+        StringModifier lowerCase = string -> ('"' + string + '"' + " -> modified: " + '"' + string.toLowerCase() + '"');
+
+        number3.transform(stringOne, lowerCase, upperCase);
+    }
+
+    //MARK: METHOD THAT ACCEPTS A STRING VALUE AND A LAMBDA EXPRESSION
```

Comment:
actually, the method `transform` needs to accept a string value and a lambda expression:

```
In the main class create a method that accepts a string value and a lambda expression,
applies the lambda to the value and outputs the result.
...
Call the defined method with some string value and both lambdas.
```